src/platform/common/platform/fs-paths.ts

```diff
@@ -6,9 +6,7 @@ export function getDisplayPath(
     workspaceFolders: readonly WorkspaceFolder[] | WorkspaceFolder[] = []
 ) {
     const relativeToHome = getDisplayPathImpl(filename);
-    const relativeToWorkspaceFolders = workspaceFolders.map((folder) =>
-        getDisplayPathImpl(filename, folder.uri.fsPath)
```

Comment:
fsPath should be illegal in non node files. I'm going to work on adding a linter rule for this.

Reply:
This is what will happen now:
![image](https://user-images.githubusercontent.com/19672699/161597502-4de86240-1086-4c0a-b654-f92225566990.png)

Reply:
That doesn't seem right though. Making it handle relative paths now.